meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -9,7 +9,7 @@ SUPERVISOR_REPOSITORY_aarch64 = "resin/aarch64-supervisor"
 SUPERVISOR_REPOSITORY_x86 = "resin/i386-supervisor"
 SUPERVISOR_REPOSITORY_x86-64 = "resin/amd64-supervisor"
 
-SUPERVISOR_TAG ?= "v6.3.6"
+SUPERVISOR_TAG ?= "v6.4.0"
```

Comment:
This commit should be removed from the PR now that https://github.com/resin-os/meta-resin/pull/896 is merged